dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/tei/query/context/querybuilder/MainTableQueryBuilder.java

```diff
@@ -45,6 +45,7 @@ public class MainTableQueryBuilder implements SqlQueryBuilder {
   @Override
   public RenderableSqlQuery buildSqlQuery(
```

Comment:
Probably we need to find a better design for it. Not in this PR, but I think these `unused` are not ideal.